student.rb

```diff
@@ -0,0 +1,15 @@
+require_relative './person'
+
+class Student < Person
+  def initialize(age, classroom, name = 'Unknown', parent_permission: true)
+    super(age, name, parent_permission: parent_permission)
```

Comment:
### Optional
- [ ] Why are we setting the parent permission to parent_permission? `parent_permission: parent_permission`

Reply:
Honestly, I didn't make it work in another way. I put it so that the `parent_permission` value from the parent-class takes the value passed to the `Student` class. If you could explain to me if this is a good approach or not I would be thankful!